controller/config/config.go

```diff
@@ -41,10 +41,10 @@ const (
 	MaxEdgeAPIActivityUpdateInterval      = 10 * time.Minute
 	MinEdgeAPIActivityUpdateInterval      = time.Millisecond
 
-	DefaultEdgeSessionTimeout = 10 * time.Minute
+	DefaultEdgeSessionTimeout = 30 * time.Minute // Don't change the time unit, keep time.Minute
 	MinEdgeSessionTimeout     = 1 * time.Minute
 
-	MinEdgeEnrollmentDuration     = 5 * time.Minute
+	MinEdgeEnrollmentDuration     = 180 * time.Minute // Don't change the time unit, keep time.Minute
```

Comment:
The comments make no sense. It isn't a big deal but I would remove them. The unit doesn't matter as long as the duration value that is generated is the desired value.

Reply:
oh - actually that's a dandy point. as long as the temaplate processes it properly the comments aren't needed. good point...

Reply:
on re-instpection, this is the `MinEdgeEnrollmentDuration` we want to change `DefaultEdgeEnrollmentDuration`... the minimum can stay at 5m

Reply:
The template does use it properly, they were converted to time.durations during the last refactoring of the default values.